app/src/examples/RuntimeTests/tests/animations/withTiming/enteringColorsSnapshots.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { StyleSheet } from 'react-native';
+import Animated, { withDelay, withTiming } from 'react-native-reanimated';
+import React from 'react';
+import {
+  describe,
+  test,
+  expect,
+  mockAnimationTimer,
+  recordAnimationUpdates,
+  render,
+  wait,
+} from '../../../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+import { Snapshots } from './entering.snapshot';
+
+const AnimatedComponent = ({ fromColor, toColor }: { fromColor: string; toColor: string }) => {
+  const customAnim = () => {
+    'worklet';
+    const animations = {
+      backgroundColor: withDelay(500, withTiming(toColor, { duration: 500 })),
+    };
+    const initialValues = { backgroundColor: fromColor };
+    return { initialValues, animations };
+  };
+
+  return <Animated.View style={styles.colorBox} entering={customAnim} />;
+};
+
+async function getSnaphotUpdates(fromColor: string, toColor: string) {
+  await mockAnimationTimer();
+  const updatesContainer = await recordAnimationUpdates();
+  await render(<AnimatedComponent fromColor={fromColor} toColor={toColor} />);
+  await wait(1200);
+  const updates = updatesContainer.getUpdates();
+  const nativeUpdates = await updatesContainer.getNativeSnapshots();
+  return [updates, nativeUpdates];
+}
+
+describe('entering with custom animation (withDelay + withTiming color changes) test', () => {
+  test.each([
+    ['rgba(16, 128, 26, 1)', 'rgba(179, 6, 6, 1)'],
+    ['rgba(143, 253, 140, 1)', 'rgba(29, 247, 107, 1)'],
+    ['rgba(226, 167, 48, 1)', 'rgba(222, 251, 137, 1)'],
+    ['rgba(201, 25, 139, 1)', 'rgba(91, 234, 32, 1)'],
+    ['rgba(246, 121, 228, 1)', 'rgba(10, 192, 235, 1)'],
+    ['rgba(12, 115, 38, 1)', 'rgba(242, 174, 57, 1)'],
+    ['rgba(169, 198, 146, 1)', 'rgba(238, 116, 102, 1)'],
+    ['rgba(222, 170, 194, 1)', 'rgba(79, 79, 145, 1)'],
+    ['rgba(116, 58, 155, 1)', 'rgba(161, 66, 77, 1)'],
+    ['rgba(232, 113, 116, 1)', 'rgba(158, 201, 27, 1)'],
+  ])('Entering color from ${0} to ${1}', async ([fromColor, toColor]) => {
+    const [updates, nativeUpdates] = await getSnaphotUpdates(fromColor, toColor);
+    const snapshotName = (fromColor + toColor)
+      .replace(/\s/g, '')
+      .replace(/[()]/g, '$')
+      .replace(/,/g, '$')
```

Comment:
It would look a bit cleaner if this was an underscore.
Now 
`16, 128, 26, 1)', 'rgba(179, 6, 6, 1)'` -> 1`6$128$26$1$$179$6$6$1$`
And it could be 
`16, 128, 26, 1)', 'rgba(179, 6, 6, 1)'` -> `$16_128_26_1$$179_6_6_1$`

Reply:
I agree!